components/bl-treeView-component/src/index.js

```diff
@@ -0,0 +1,139 @@
+import { useState, useEffect, useCallback } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function TreeView({ component, eventHandlers }) {
+  const { display, style, classList, data, space } = component;
```

Comment:
Try to avoid names which doesn't say anything.

data - it's absolutely anything and nothing at the same time. 

treeItems | unhandledTree | unprocessedTree | itemsToValidate

Please, spend a couple of minutes when choosing variable name - that could save a lot of time when you returns to this code again.